test/bonjour.js

```diff
@@ -135,6 +151,7 @@ test('bonjour.findOne - emitter', function (bonjour, t) {
     var browser = bonjour.findOne({ type: 'test' })
     browser.on('up', function (s) {
       t.equal(s.name, 'Emitter')
+      browser.stop()
```

Comment:
How come you added the `browser.stop()` calls a few places? Did the event loop hang?